styles/nav.scss

```diff
@@ -119,7 +119,7 @@ $daptiv-subnav-tab-h-pad: 14px;
         }
     }
 
-    .daptiv-nav-menuitem-alternate {
+    .daptiv-nav-menuitem.alternate {
```

Comment:
ah sorry, i only read half of the previous rule, thought it was subnav only. Position is :+1: